README.md

```diff
@@ -108,7 +108,10 @@ pid: (INT) process ID of the created process
 
 pid: (NUM) Process ID of the subprocess that exit
 
-exitCode: (NUM) exit code of the subprocess that exit
+exitCode: (NUM) exit code of the subprocess that exit. 
+    0-255: Exited via exit(), exit code is exitCode
+    256: Exit code or signal could not be determined
+    257-inf: Exited by being kill()'d. Signal is (exitCode - 256)
```

Comment:
```suggestion
    257-inf: Exited by being kill()'d. Signal is (256 + exitCode )
```

Reply:
Its the other way around tho, if the exitCode parameter is 269, the signal is (269 - 256) ie 13 ie SIGTERM

Reply:
I didn't read the sentence, I thought the comment was about the functionality